### PR TITLE
rename sqlflow_submitter to runtime

### LINKED ISSUE #2603: Rename sqlflow_submitters to sqlflow.runtime
As the discussion of #2494 , `sqlflow_submitters` is actually the runtime of the executables of SQLFlow. We should make the name of the package fit its functionality.

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed #2603

### CODE REVIEW

File: doc/design/ant_xgboost.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ USING sqlflow_models.xgboost_model_table;
 As `codegen.go` generating TensorFlow code from sqlflow AST,
 we will add `codegen_xgboost.go` which translate sqlflow AST into a python launcher program of xgboost. 
 
-Since xgblauncher provide `DataSource` and `ModelSource`, abstraction of custom I/O pipeline, by which we can reuse data/model pipeline of `sqlflow_submitter`.
+Since xgblauncher provide `DataSource` and `ModelSource`, abstraction of custom I/O pipeline, by which we can reuse data/model pipeline of `sqlflow_runtime`.
```
Comment (wangkuiyi) [inline doc/design/ant_xgboost.md]: If the Ant-XGBoost codegen is obsolete, how about we remove this document in a future PR?

File: doc/design/ant_xgboost.md [hunk 2]
```diff
@@ -127,10 +127,10 @@ With the help of xgblauncher, we can launch xgboost from sqlflow AST via a light
 The `code template` roughly includes components as follows: 
 
 * `TFDataSource` that is responsible for fetching and pre-processing data via tf.feature_columns API.
-   Data will be fetched in mini-batch style by executing TF compute graph with mini-batch data feed by `sqlflow_submitter.db.db_generator`.
+   Data will be fetched in mini-batch style by executing TF compute graph with mini-batch data feed by `sqlflow_runtime.db.db_generator`.
 
 * `DBDataSource` that is responsible for writing prediction results into specific data base.
-   The writing action can be implemented via `sqlflow_submitter.db.insert_values`.
+   The writing action can be implemented via `sqlflow_runtime.db.insert_values`.
```
Comment (wangkuiyi) [inline doc/design/ant_xgboost.md]: Should `sqlflow_runtime` be `sqlflow.runtime`?

File: doc/design/ant_xgboost.md (hunk 1 above)
Comment (Yancey0623) [reply doc/design/ant_xgboost.md]: Sure, would remove it in the next PR.

File: doc/design/ant_xgboost.md (hunk 2 above)
Comment (Yancey0623) [reply doc/design/ant_xgboost.md]: The package name `sqlflow` is already used by Python client: https://github.com/sql-machine-learning/pysqlflow , I will remove the prefix `sqlflow_`.

**Review by wangkuiyi — APPROVED** [review]
LGTM+

**Pull request merged**